bundles/org.openhab.binding.atlona/src/main/resources/OH-INF/i18n/atlona.properties

```diff
@@ -73,12 +79,6 @@ thing-type.atlona.pro3-1616m.group.volume8.label = Volume 8
 thing-type.atlona.pro3-1616m.group.volume8.description = Volume 8 channels
 thing-type.atlona.pro3-1616m.group.volume9.label = Volume 9
 thing-type.atlona.pro3-1616m.group.volume9.description = Volume 9 channels
-thing-type.atlona.pro3-1616m.group.volume10.label = Volume 10
```

Comment:
This re-ordering of atlona.properties is not necessary.

Reply:
This is what the i18n plug-in does …

Reply:
Some of these were arranged in numerical or some other logical order that matches the xml files. The i18n plug in is re-ordering them into alphabetical order.

Reply:
But it makes much more sense to have it in the order the plugin creates. For future updates most likely the plugin will be used and then it is easier to see the differences.